app/browser/zoom.js

```diff
@@ -10,11 +17,28 @@ exports = module.exports = () => {
 		}
 
 		if (event.ctrlKey) {
-			if (keyName === '+') {
-				webFrame.setZoomLevel(webFrame.getZoomLevel() + 1);
-			} else if (keyName === '-') {
-				webFrame.setZoomLevel(webFrame.getZoomLevel() - 1);
-			}
+			setNextZoomLevel(keyName, config);
 		}
 	}, false);
-};
\ No newline at end of file
+};
+
+function restoreZoomLevel(config) {
+	ipcRenderer.invoke('getZoomLevel', config.partition).then(zl => {
+		webFrame.setZoomLevel(zl);
+	});
+}
+
+function setNextZoomLevel(keyName, config) {
+	const zoomFactor = zoomLevels[keyName];
+	var zoomLevel = webFrame.getZoomLevel();
+	if (typeof (zoomFactor) !== 'number') {
+		return;
```

Comment:
Probably worth adding a Log message in here.

Reply:
Is it to log the current zoom level?